apps/server/src/shared/domain/entity/cardElement.entity.ts

```diff
@@ -18,20 +17,6 @@ export abstract class CardElement extends BaseEntityWithTimestamps {
 	cardElementType!: CardElementType;
 }
 
```

Comment:
side note: should we maybe rename this file to `card-element.entity.ts` to be consistent ?

Reply:
I definitely agree. However, some files are named in this style. I can't find any rule about this in the documentation, Is there one?

Reply:
@EzzatOmar @virgilchiriac 
what do you say about it?

Reply:
makes sense, yes